pkg/aws/install.go

```diff
@@ -364,15 +319,29 @@ func InstallCluster(name, releaseImage, dhParamsFile string, waitForReady bool)
 		return fmt.Errorf("cluster pod CIDR exceeds max address space")
 	}
 
+	apiDNSName, err := waitForServiceLoadBalancerDNS(client, name, kubeAPIServerServiceName)
```

Comment:
s/waitForServiceLoadBalancerDNS/waitForServiceLoadBalancerAddress/
In GCP and Azure you'll get an IP and not a DNS name